bubblesub/ui/audio/base.py

```diff
@@ -293,6 +294,9 @@ def pts_from_x(self, x: float) -> int:
         scale = self._view.view_size / self.width()
         return int(x * scale + self._view.view_start)
 
+    def _get_paint_cache_key(self) -> int:
+        pass
```

Comment:
At least reraise `NotImplementedError` rather than returning `None` in a function that declares itself to always return an `int`